transformer_engine/common/normalization/common.cpp

```diff
@@ -424,5 +416,8 @@ NormalizationPlanBase* NormalizationPlanRegistry::getNormalizationPlan(
   return normalizationPlanMap[key].get();
 }
 
+bool use_cudnn_norm_fwd() { return transformer_engine::getenv<bool>("NVTE_NORM_FWD_USE_CUDNN"); }
+bool use_cudnn_norm_bwd() { return transformer_engine::getenv<bool>("NVTE_NORM_BWD_USE_CUDNN"); }
```

Comment:
This approach is fine for now, but we'll eventually want to avoid the CPU overhead of checking envvars every time we call these functions. Maybe something like:
```c++

namespace {

bool& cudnn_norm_fwd_flag() {
  static bool flag = transformer_engine::getenv<bool>("NVTE_NORM_FWD_USE_CUDNN");
  return flag;
}

bool& cudnn_norm_bwd_flag() {
  static bool flag = transformer_engine::getenv<bool>("NVTE_NORM_BWD_USE_CUDNN");
  return flag;
}

}  // namespace

bool cudnn_norm_fwd_is_enabled() { return cudnn_norm_fwd_flag(); }
bool cudnn_norm_bwd_is_enabled() { return cudnn_norm_bwd_flag(); }
void enable_cudnn_norm_fwd(bool enable) { cudnn_norm_fwd_flag() = enable; }
void enable_cudnn_norm_bwd(bool enable) { cudnn_norm_bwd_flag() = enable; }
```
This also exposes a way to enable/disable cuDNN in the tests without messing with the environment.

Reply:
Good to learn that we can override the static flag this way! 
I changed the code and test accordingly.